Ghost filter/Ghost filter code.py

```diff
@@ -0,0 +1,35 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import cv2
+# take path of the image as input
+img_path = input()  #example -> C:\Users\xyz\OneDrive\Desktop\project\image.jpg  
```

Comment:
Give a print statement for this, for example:
```python
num = int(input("Enter a num: "))
```

Reply:
done!